test/CSSTransition-test.js

```diff
@@ -237,10 +266,12 @@ describe('CSSTransition', () => {
           expect(nodeRef.current.className).toEqual(
             'not-appear-test-enter-done'
           );
-          expect(count).toEqual(2);
-          done();
```

Comment:
Should we add `++count` and verify it in the later `waitFor` callback to confirm this `onEntered` was called? Otherwise, this test would be passed even if the `onEntered` hasn't been called.

Reply:
These checks and the ones mentioned later all moved further down in the test e.g.
```ts
await waitFor(() => {
    expect(count).toEqual(2);
});
```

Does that make sense?

Reply:
Yeah, that makes sense  for me 👍  Thank you!